ibodata/depth_profile.py

```diff
@@ -1,6 +1,36 @@
+import numpy as np
+
 from beprof.profile import Profile
 
 
 class DepthProfile(Profile):
-    """TODO"""
-    pass
+
+    def range(self, level):
+        return self.width(level)
+
+    def FWHM(self):
+        return self.width(0.5)
+
+    def max_plat_ratio(self):
+        return self.y.max()/self.y[self.x.min()]
```

Comment:
indexing by x won't work this way.
Why not `self.y[0]` ?

Reply:
Fixed